birdie/src/lib/components/files/FileTree.svelte

```diff
@@ -18,13 +18,15 @@
 	const level = 0;
 
 	const handleSaveFileTree = async () => {
-		await fs.writeFile(FILE_TREE_PATH, JSON.stringify(fileNode, null, 2), {
-			dir: fs.BaseDirectory.AppLocalData
+		await fs.writeTextFile(FILE_TREE_PATH, JSON.stringify(fileNode, null, 2), {
```

Comment:
a bunch of issues were related to attempting to write a string using `writeFile`, which as of tauri2 accepts a byte stream instead of a string